twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/ui/main/MainViewModel.kt

```diff
@@ -19,18 +23,21 @@ import java.lang.Exception
  * Main ViewModel
  * @property view the instance of the view
  */
-class MainViewModel(myApplication: Application) : AndroidViewModel(myApplication) {
+class MainViewModel(val myApplication: Application) : AndroidViewModel(myApplication) {
```

Comment:
This property is actually needed? There is no way to access a `context` within the `AndroidViewModel`?

Reply:
There is, I think you just need to call `getApplication()`